samples/nrf9160/fmfu_smp_svr/src/main.c

```diff
@@ -7,9 +7,41 @@
 #include <zephyr.h>
 #include <mgmt/fmfu_mgmt.h>
 #include <mgmt/fmfu_mgmt_stat.h>
+#include <nrf_modem_full_dfu.h>
 #include <modem/modem_info.h>
 #include <modem/nrf_modem_lib.h>
 #include <modem/at_cmd.h>
+#ifdef CONFIG_MCUMGR_CMD_IMG_MGMT
+#include "img_mgmt/img_mgmt.h"
+#endif
+#ifdef CONFIG_MCUMGR_CMD_OS_MGMT
+#include "os_mgmt/os_mgmt.h"
+#endif
+
+static void fmfu_callback_handler(const enum fmfu_evt_id id)
```

Comment:
Is there any point in having `id` const? As its not a pointer I mean. Also it does not seem to match the function signature from the header file.

Reply:
No point will change it was a pointer in my initial idea.